girder/cli/mount.py

```diff
@@ -96,7 +96,7 @@ def _configure_disk_cache(self, cacheopts):
         for key in list((cacheopts or {}).keys()):
             if key.startswith('diskcache'):
                 value = cacheopts.pop(key)
-                if value.strip().isdigit():
+                if str(value).strip().isdigit():
```

Comment:
It'd be nice to use a `dataclass` for the `cacheopts` argument, and if any of the options are algebraic sum types, we could represent them with their own special types.

Reply:
I didn't want to re-enumerate the possible options for diskcache.  If diskcache had such a dataclass, that would be great.

Reply:
Looking at diskcache's source, it looks like `diskcache.core.DEFAULT_SETTINGS` plus `directory` and `timeout` is the list of parameters we could possible deal with.  But they aren't explicitly typed, so you'd be doing type inspection for those to determine what we should cast to...

Reply:
I'll merge this for now; I may try to refactor to be smarter about this on a future PR.

Reply:
Sure thing, don't mean to hold this up, was just musing :)